services/src/main/java/org/keycloak/authentication/authenticators/broker/IdpEmailVerificationAuthenticator.java

```diff
@@ -115,6 +115,8 @@ private void sendVerifyEmail(KeycloakSession session, AuthenticationFlowContext
         RealmModel realm = session.getContext().getRealm();
         UriInfo uriInfo = session.getContext().getUri();
         AuthenticationSessionModel authSession = context.getAuthenticationSession();
+        // TODO: split to a new PR
+        authSession.addRequiredAction(UserModel.RequiredAction.VERIFY_EMAIL);
```

Comment:
Is it correct, that this should be now removed from this PR since we already have https://github.com/keycloak/keycloak/commit/142c1413 ?